addons/l10n_it_edi_ndd/models/account_move.py

```diff
@@ -1,5 +1,6 @@
-from odoo.addons.l10n_it_edi_ndd.models.account_payment_methode_line import L10N_IT_PAYMENT_METHOD_SELECTION
 from odoo import api, fields, models
+from odoo.tools.sql import column_exists, create_column
+from odoo.addons.l10n_it_edi_ndd.models.account_payment_methode_line import L10N_IT_PAYMENT_METHOD_SELECTION
```

Comment:
Sorry for this but, could you swap them ? Alphabetical order and all :sweat_smile:

Reply:
You mean swap the two lines?
From what I can see in the documentation, that's the right order: https://www.odoo.com/documentation/18.0/contributing/development/coding_guidelines.html#imports, no ? 😅

Reply:
damnnn i just learned something, never seen this applied i believe :joy: didn't know `odoo.addons` had their own section, thank you :+1: